calculator.py

```
@@ -16,23 +16,100 @@ def divide(a, b):
+
+    male_endings = ['й', 'н', 'л', 'р', 'м', 'в', 'г', 'д', 'к', 'т', 'ий', 'им', 'ан', 'он']
+    female_endings = ['а', 'я', 'на', 'ия', 'ла']
+    neutral_endings = ['и', 'с', 'ь', 'о', 'е', 'ен', 'ис']
+
+    name = name.strip().lower()
+
+    if not name:
+        return random.choice(genders)
+
+    # 30% шанс получить небинарный/другой гендер
+    if random.random() < 0.3:
```

Ruff (0.13.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)